client/src/components/forms/FormButton/FormButton.module.less

```diff
@@ -1,9 +1,9 @@
-@import "less/variables.less";
+@import 'less/variables.less';
 
 .submitBtn {
   align-self: start;
   height: 2rem;
-  border-radius: 0.125rem;
+  border-radius: 0.125rem !important;
```

Comment:
Something seems to be wrong with these border-radiuses - i will try to check it tomorrow

Reply:
Ok, seems to be the quickest way to override ant-design styles